telegram/ext/jobqueue.py

```diff
@@ -252,9 +217,8 @@ def run_monthly(self, callback, when, day, context=None, name=None, day_is_stric
                 Can be accessed through ``job.context`` in the callback. Defaults to ``None``.
             name (:obj:`str`, optional): The name of the new job. Defaults to
                 ``callback.__name__``.
-            day_is_strict (:obj:`bool`, optional): A flag to check if the monthly job should
-                respect the day specified. This flag should only be set to ``False`` if you
-                want to schedule the job on the end of every month. Defaults to ``True``.
+            day_is_strict (:obj:`bool`, optional): If ``False`` and day > month.days, will pick
```

Comment:
Thanks. Please remove the note below since it contains no additional information.

Reply:
done!